module/VuFind/src/VuFind/Db/Entity/ShortlinksEntityInterface.php

```diff
@@ -40,4 +42,58 @@
  */
 interface ShortlinksEntityInterface extends EntityInterface
 {
+    /**
+     * Get identifier (returns null for an uninitialized or non-persisted object).
+     *
+     * @return ?int
+     */
+    public function getId(): ?int;
+
+    /**
+     * Get shortened URL.
+     *
+     * @return string
+     */
+    public function getPath(): string;
+
+    /**
+     * Set shortened URL.
```

Comment:
I reviewed the code as we discussed, and this is indeed always going to be a path, not a full URL. So I think the comment could be something like:

Set the path (e.g. /Search/Results?lookfor=foo) of the URL being shortened; shortened URLs are always assumed to be within the hostname where VuFind is running.